examples/clone/clone.bal

```diff
@@ -0,0 +1,69 @@
+import ballerina/io;
+
+//Define anydata record
+public type Person record {
+    string name;
+    int age;
+    boolean married;
+    float salary;
+    Address address;
+};
+
+public type Address record {
+    string country;
+    string state;
+    string city;
+    string street;
+};
+
+// The returned tuple will be later used by ballerina tests
+public function main() returns (Person, Person, string)? {
+    // Create a value of Address
+    Address address = {
+        country : "USA",
+        state: "NC",
+        city: "Raleigh",
+        street: "Daniels St"
+    };
+
+    // Create a value of type Person
+    Person person = {
+        name: "Alex",
+        age: 24,
+        married: false,
+        salary: 8000.0,
+        address: address
+    };
+
+    // Note that we have assigned the person value to any typed variable. So the type of the value has to be determined
+    // in runtime.
+    any anyValue = person;
+
+    // Invoke the clone builtin method. The return type is either Person or error. Note that the error is returned if
+    // the value being cloned is not of anydata type. If the type of the value being cloned can be determined in
```

Comment:
Add "anydata" between single back ticks. Apply to all such instances and similar values in the example.

Reply:
Fixed